handlers/announcement.go

```diff
@@ -195,12 +195,11 @@ func insertNewAnnouncement(db *sql.DB) {
 	ctx := context.Background()
 	var announcements = scrapeAnnouncements()
     for i := range announcements {
-        strings.Replace(announcements[i].Date,".","-",-1)
-        strings.Replace(announcements[i].Date,"/","-",-1)
-        format := "2023-01-23"
-        realdate, _ := time.Parse(announcements[i].Date, format)
+        announcements[i].Date =  strings.Replace(announcements[i].Date,".","-",-1)
+        announcements[i].Date = strings.Replace(announcements[i].Date,"/","-",-1)
+        t, _ := time.Parse("02-01-2006", announcements[i].Date)
```

Comment:
is this DD-MM-YYYY or DD-MM-YYYY?

Reply:
What? Didn't understand your question.

The dates on the official website are in DD-MM-YYYY, or DD.MM.YYYY or DD/MM/YYYY format, so this code tries to handle those cases and convert to DD-MM-YYYY

Reply:
"02-01-2006" is "DD-MM-YYYY" or "MM-DD-YYYY"
better idea to use a date >12

Reply:
I looked this format string up, apparently in Go the format is always given in terms of a single date 2/1/2006 (DD/MM/YYYY) so this problem does not arise.